### PR TITLE
Rename pyscript-cli -> pyscript.

### PR DESCRIPTION
**Pull request by ntoll**
This PR introduces a single change. The package name is changed from `pyscript-cli` to `pyscript`.

**TODO**: Register `pyscript` on PyPI and remove `pyscript-cli`.

### CODE REVIEW

**Review by fpliger — APPROVED** [review]
LGTM. I'm checking to actually have an email like team@pyscript.net or something like this to use as authors... 

Imho, it's good to merge and upload to PyPI

File: pyproject.toml [hunk 1]
```diff
@@ -3,12 +3,16 @@ requires = ["poetry-core>=1.0.0"]
...
 description = "Command Line Interface for PyScript"
-authors = ["Matt Kramer <mkramer@anaconda.com>"]
+authors = [
+    "Matt Kramer <mkramer@anaconda.com>",
+    "Fabio Pliger <fpliger@anaconda.com>",
+    "Nicholas Tollervey <ntollervey@anaconda.com>",
+]
 license = "Apache-2.0"
-repository = "https://github.com/mattkram/pyscript-cli"
+repository = "https://github.com/pyscript/pyscript-cli"
```
Comment (mattkram) [inline pyproject.toml:15]: Thanks 👍

**Comment by mattkram** [comment]
> LGTM. I'm checking to actually have an email like [team@pyscript.net](mailto:team@pyscript.net) or something like this to use as authors...
> 
> Imho, it's good to merge and upload to PyPI

This LGTM too. We will need to do some config in the settings to get the token updated and such. Will discuss w/ you offline about best approach there.

**Review by mattkram — APPROVED** [review]
🚀

**Comment by ntoll** [comment]
https://pypi.org/project/pyscript/

Now on PyPI. Thanks @jezdez for the help in making this happen.

**Pull request merged**